src/Autofac.BindingByConvention/Autofac.BindingByConvention.csproj

```diff
@@ -16,7 +16,7 @@
     <NeutralLanguage>en</NeutralLanguage>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <AssemblyVersion>0.0.3</AssemblyVersion>
+    <AssemblyVersion>0.0.5</AssemblyVersion>
```

Comment:
upgrade is needed ? Deployed NuGet is still at 0.0.1. and here we target the develop branch